### DISCUSSION TITLE
How do I delete the _prisma_migrations table?

### DISCUSSION THREAD

**Discussion by thedevbrian1** [prisma/prisma#17745] [Q&A] [answered]
I'm using prisma and deploying it to fly.io. I deleted the migrations in the prisma/migrations folder. When I run prisma migrate deploy it still applies the migrations that I deleted. How do I delete the _prisma_migrations table in order to completely get rid of the deleted migrations?

**Comment by thedevbrian1** [comment, marked as answer]
Use the sqlite3 terminal. Open the terminal by using the command `sqlite3 ` Then use the  `DROP TABLE _prisma_migrations;` command

**Reply by nurul3101** [reply]
Had you deleted all the migrations from `prisma/migrations` folder or only specific ones?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
